### ISSUE TITLE
Is it possible to emulate a second monitor with 2 laptops?

### ISSUE THREAD

**Issue by ghost** [input-leap/input-leap#798]
> This issue has been migrated from old Barrier Github repository https://github.com/debauchee/barrier/issues/798
> 
> Issue created on: 2020-07-16 by @igorcmelo
> Issue last updated on: 2020-09-25

# I have two laptops and I would like to emulate as if one was a second monitor. 
My goal is to extend screen for video editing, like this example picture:

![Example picture](https://external-content.duckduckgo.com/iu/?u=http%3A%2F%2Fwww.dvinfo.net%2Fforum%2Fattachments%2Fadobe-creative-suite%2F31700d1391612071-how-do-you-setup-your-monitors-3-screen-editing-rimg0029.jpg&f=1&nofb=1)

---

> Commented on: 2020-07-22 by @Acters

This program allows mouse and keyboard to control both computers without having to physically switch the USB between two computers. It also has the added benefit of file sharing via drag and drop, and clipboard sharing(copy + paste) between two SEPARATE computers.

If you want to do this then you need something else. there is no way to share laptop power between each other. 
Extending a screen from one laptop to another laptop will make only one laptop work "like" a monitor but not help with any of the work. It will negatively affect performance because only ONE laptop will do ALL the work while the other one just makes pretty pictures. On top of that, The one laptop doing all the work will have to also work harder to simulate, communicate, AND synchronize with the other laptop that is just a monitor. this is highly inefficient.
I highly recommend getting a monitor. they are cheap to get and easier to get working than what you are asking for.(without all these negatives mentioned.)

This program, instead, allows you to separate different tasks between separate computers or manage a system of computers that each work separately WITH ONLY A KEYBOARD AND MOUSE. It's a keyboard and mouse extender. That's it. It is up to you to find a way to make a proper workflow. This program also has the "quality of life" features of file sharing with drag and drop, and clipboard sharing(copy + paste).

In short, no. This is mouse and keyboard extending only. There is no monitor or usb(like external drives) extending features. This does have features like file sharing and clipboard sharing(copy + paste) to make our lives easier.

---

> Commented on: 2020-09-21 by @github-actions[bot]

This issue has been automatically marked as stale due to inactivity. It will be closed if no further activity occurs. Thank you for your contributions.

---

> Commented on: 2020-09-25 by @shymega

Yes, perfectly articulated. We cannot support this, only mouse and keyboard extending. If you want a proper screen sharer, maybe use a physical KVM switch?

Thanks, closing.

---

> Closed on: 2020-09-25 by @shymega
